includes/Integrations/Site_Kit.php

```diff
@@ -190,13 +221,16 @@ public function get_plugin_status() {
 					'plugin-install.php'
 				)
 			);
+			$adsense_link   = $analytics_link;
 		}
```

Comment:
Nit-pick, but I think working with early returns here instead of the complex if-else chains would simplify the logic. Also right now, a lot of logic is potentially run unnecessarily, e.g. if the whole Site Kit plugin is not active, it's redundant to check whether Analytics and AdSense modules are active.

Reply:
Much of this complex if else came from code already found in the codebase, I just tweaked it. For early returns, we would need to break this logic into another function.

Reply:
Sounds like an opportunity to make the code a bit more readable 👍